solutions/standard/main.tf

```diff
@@ -66,9 +66,9 @@ module "secrets_manager" {
   kms_key_crn                       = local.kms_key_crn
   skip_kms_iam_authorization_policy = var.skip_kms_iam_authorization_policy
   # event notifications dependency
-  enable_event_notification        = var.existing_en_instance_crn != null ? true : false
-  existing_en_instance_crn         = var.existing_en_instance_crn
-  skip_en_iam_authorization_policy = var.skip_en_iam_authorization_policy
+  enable_event_notification        = var.existing_event_notification_instance_crn != null ? true : false
+  existing_en_instance_crn         = var.existing_event_notification_instance_crn
```

Comment:
do we want to clean up these variable names while we're making this change?

Reply:
@alex-reiff , we are just removing the acronyms as most consumers, in particular DA consumers, won't understand easily the acronyms.